lib/cli/commands/build.js

```diff
@@ -121,6 +121,20 @@ build.builder = function(cli) {
 			default: false,
 			type: "boolean"
 		})
+		.option("output-style", {
+			describe:
+				"Processes build results into a specific directory structure. " +
+				"\"Flat\" omits the project namespace and the \"resources\" directory. This is" +
+				" currently only supported for projects of type 'library'. Projects of " +
+				"type 'application' always result in a flat output." +
+				"No other dependencies can be included in the build result.",
```

Comment:
Other dependencies can be included now

Reply:
Thanks!
I'll align this with the table!